tests/cli/test_family.py

```diff
@@ -80,6 +83,30 @@ def test_family_cutoffs_set_unit(run_cli_command, get_pseudo_family, generate_cu
     assert family.get_cutoffs_unit() == unit
 
 
+@pytest.mark.parametrize(
+    'family_cls,label', [(SsspFamily, 'SSSP/1.1/PBE/efficiency'),
+                         (PseudoDojoFamily, 'PseudoDojo/0.4/PBE/SR/standard/psp8')]
+)
+@pytest.mark.usefixtures('clear_db')
+def test_family_cutoffs_set_established(
```

Comment:
Is there still a test for a family type that is actually accepted?

Reply:
Yes, `test_family_cutoffs_set` is still there!